build-system/amp.extern.js

```diff
@@ -123,6 +123,8 @@ AmpConfigType.prototype.localDev;
 AmpConfigType.prototype.v;
 /* @public {boolean} */
 AmpConfigType.prototype.canary;
+/* @public {string} */
+AmpConfigType.prototype.runtime;
```

Comment:
where will runtime get used?

Reply:
It was just missing. Already being used.